app/assets/stylesheets/_lists.scss

```diff
@@ -1,3 +1,21 @@
+%default-sub-sublist{
+    ol{
+      list-style-type: lower-roman;
+    }
+    ul{
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Opening curly brace `{` should be preceded by one space